portfolio/src/main/webapp/script.js

```diff
@@ -66,13 +66,16 @@ function getComments() {
             console.log('Comments: ', comments);
             bodyElement.innerHTML += 'Here are all the comments: <br>';
             for (comment of comments) {
-                bodyElement.innerHTML += comment.propertyMap.text + '<br>';
+                bodyElement.innerHTML += comment + '<br>';
             }
 	});
 }
 
-function deleteComments() {
-    console.log("Starting to delete comments!");
-    fetch('/delete-data');
-    console.log("Complete!");
+/*
+ * Creates a map and adds it to the page. 
+ */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
```

Comment:
Is this related to this PR?

Reply:
this is merged in from the master branch

Reply:
should I delete it?

Reply:
oh I'll try updating Week-6-1 to see if that helps!